pages/data/zarr-access.md

```diff
@@ -0,0 +1,23 @@
+import { Link } from '@carbonplan/components'
+import Section from '../../components/section'
+
+# Zarr
+
+We prefer the Zarr data format for storing our raster data products because it's simple, transparent, open, community-driven, and optimized for the cloud.
```

Comment:
```suggestion
We prefer the Zarr data format for storing our raster data products because it's simple, open, community-driven, and optimized for the cloud.
```